docs/best-practices/app-structure-breakdown.md

```diff
@@ -170,7 +170,7 @@ Your app will ultimately perform some logic based on how the user interacts with
 
 ### Resolver Functions
 
-When the user interacts with the app and updates the values of the `Variable`s in your app, this will update `DerivedVariable`s and `DataDerivedVariable`s or trigger actions like `dara.core.interactivity.actions.UpdateVariable` and `dara.core.interactivity.actions.SideEffect`. What these have in common is they take a resolver function that defines how to update based on the new values of the `Variable`s.
+When the user interacts with the app and updates the values of the `Variable`s in your app, this will update `DerivedVariable`s and `DataDerivedVariable`s or trigger actions like `dara.core.interactivity.actions.
```

Comment:
I don't think we need the `dara.core.interactivity.actions` link anymore, probably just link to the actions docs page instead?

Reply:
good point